queries.sql

```diff
@@ -7,4 +7,40 @@ SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered=TRUE;
 SELECT * FROM animals WHERE NOT name='Gabumon';
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 and 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 and 17.3;
+
+BEGIN TRANSACTION
+UPDATE  animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN TRANSACTION;
+UPDATE animals SET species = 'digimon' WHERE NAME LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE NAME  NOT LIKE '%mon';
+COMMIT;
+SELECT * FROM animals;
```

Comment:
- [ ] Good job working on this query, but please kindly verify the change was made before committing.